conf/gravitino.conf.template

```diff
@@ -39,6 +39,8 @@ gravitino.server.webserver.threadPoolWorkQueueSize = 100
 gravitino.server.webserver.requestHeaderSize = 131072
 # The response header size of the built-in web server
 gravitino.server.webserver.responseHeaderSize = 131072
+# Comma separated list of REST API packages to scan
+gravitino.restApiPackages = org.apache.gravitino.server.web.rest
```

Comment:
Will this affect the iceberg rest server, did you test it, @FANNG1 please also verify it.

Reply:
Besides, I think this should not be a configuration for users. "org.apache.gravitino.server.web.rest" should always be configured, right? What is the behavior if users configure others?

You'd better configure like "additionalPackages" to support additional rest packages, and make Gravitino's package as a always loaded package.

Reply:
If we use configure like "additionalPackages" that means we don't allow user to overwrite the existing REST APIs?

Reply:
Do you have such requirements to overwrite? I don't think people have such requirements.

Reply:
OK, maybe we can make it `additionalPackages` and support overwrite when necessary

Reply:
updated